docs/sources/mimir/reference-configuration-parameters/index.md

```diff
@@ -856,6 +856,10 @@ instance_limits:
 # the -ingester.max-global-series-per-user limit.
 # CLI flag: -ingester.ignore-series-limit-for-metric-names
 [ignore_series_limit_for_metric_names: <string> | default = ""]
+
+# (advanced) Retention of ephemeral series.
+# CLI flag: -ingester.ephemeral-series-retention-period
+[ephemeral_series_retention_period: <duration> | default = 10m]
```

Comment:
I would have expected to see this close to `-blocks-storage.tsdb.retention-period`. Have you considered having something like `-blocks-storage.ephemeral-tsdb.retention-period`?

Reply:
Funny that you ask. I originally started with similar flag, but it then found metadata retention and copied that instead. I don't have strong opinion about naming, I will switch to `-blocks-storage.ephemeral-tsdb.retention-period`.